api/LargestContentfulPaint.json

```diff
@@ -159,13 +159,22 @@
             "web-features:largest-contentful-paint"
           ],
           "support": {
-            "chrome": {
-              "version_added": "77"
-            },
+            "chrome": [
+              {
+                "version_added": "133"
+              },
+              {
+                "version_added": "77",
+                "partial_implementation": true,
+                "notes": "Returns 0 for cross-origin media."
```

Comment:
Whenever we have multiple browsers with the same notes, I feel like it would be better to add a behavioral subfeature for it.

@Elchi3 Wdyt?

Reply:
Yes, this seems like a new spec requirement and a sub feature to track it would be good. That way web-features can also handle this better.

Reply:
@tunetheweb Can you make this change for us? The idea is to add a subfeature, e.g. `cross-origin_media` where Chrome has version_added in 133, and you can keep the parent feature as is, i.e. version_added in 77 without limitations.

Reply:
Done. I went with `cross-origin-render-time` as think thats' more accurate reflects what this is.

It also reflects how it was documented in MDN: https://developer.mozilla.org/en-US/docs/Web/API/LargestContentfulPaint#cross-origin_image_render_time (it does include "image" but think that's excessive and there's changes to include video so leaving that off the attribute here).